ddtrace.gemspec

```diff
@@ -40,6 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'opentracing', '>= 0.4.1'
 
   # Development dependencies
+  spec.add_development_dependency 'rack'
```

Comment:
We should not need to add a development dependency here. I presume you might have been trying to run the tests and found Rack wasn't available? If so I apologize for the confusion, as we haven't yet documented how to do so. In essence, you would want to:

```
# Install dependencies for gem
bundle install
# Install dependencies for each test configuration
appraisal install
# Run RSpec tests for Rack using the "contrib" test configuration (which contains Rack)
bundle exec appraisal contrib rake spec:rack
```

Reply:
Good to know, I'll remove